### ISSUE TITLE
Stuck at 0% Queue

### ISSUE THREAD

**Issue by souldessin** [audapolis/audapolis#394]
Hello, this looks very promising.

I wanted to try it out on my system and downloaded the AppImage. I ran it and downloaded the large English pack and then tried to import & transcribe. It converted it and then just stayed at 0% on the progress bar, saying it was queued. This stayed in this state for the entire evening. I later tried the .deb file and installed it. It found the already downloaded English pack and proceeded to do the same. I'm not sure what to do beyond close and retry. there wasn't even a back option.

System | Value
-|-
OS | Kubuntu 22.04.1 LTS
Audapolis Version | 0.2.1
First Attempt | AppImage
Second Attempt | DEB package
Language Pack | English (Big)
File attempted | [The MP3 from this page, can't upload an MP3](https://lifespringmedia.com/family-bible/romans-1-2-an-airtight-case/)

**Comment by pajowu** [comment]
Oh no /o\. Thank you for reporting this bug. I'm currently working on fixing another issue (#395). Once that is done, I can release a version with better debugging tools which would hopefully enable me to debug your issue. Would you be willing in helping me with that once this is possible? (Should just be sending me 1 or 2 debug logs)

**Comment by souldessin** [comment]
Good afternoon,

I'd be happy to help. Just comment here when you have that version ready and I'll check everything.

Thanks for the good work.

**Comment by pajowu** [comment]
I just release v0.2.2-pre3, which includes the fixes I mentioned. Please try again with that version. If the issue persists, please send me the debug log. You can export it by clicking on the main menu -> "Help" -> "Export debug log". You can either upload this file here into the issue or send it to us via the email on https://github.com/audapolis

**Comment by schmidthausen** [comment]
Hello, I just tried audapolis and I experienced the same bug. I ran top while trying it out and audapolis keeps using more and more memory and swap until the oom-killer steps in.
[debug_log.zip](https://github.com/audapolis/audapolis/files/9826488/debug_log.zip)

**Comment by pajowu** [comment]
Thank you for the debug log. From the log it seems as if the backend successfully loaded the model and even started transcribing. Are you sure it was stuck at "0% - queued"?

**Comment by schmidthausen** [comment]
The gui said stuck at 0% queue. I watched "top" as it was processing. As I said already, during transcription, memory usage kept climbing until I almost ran out and the oom killer stopped the audapolis processes. It left the GUI window open though. I should do a trace later today/tonight to give you more info.

Oct 20, 2022 01:20:50 pajowu ***@***.***>:

**Comment by pajowu** [comment]
Okay, that might be another issue. I opened #404 for that as well. Seems like we should warn users if the free memory is less than what the models need (especially the big models are indeed quite big)
